cdap-ui/app/features/hydratorplusplus/controllers/create/partials/postactions-ctrl.js

```diff
@@ -56,6 +58,30 @@ class HydratorPlusPlusPostActionsCtrl {
     this.activeTab = 'OVERVIEW';
   }
 
+  /*
+   * Because there is no ability to change version for post-run actions in UI,
+   * this function is intended to show only 1 version. The assumption is that
+   * the list the backend sends, the latest version of the plugin is the one last
+   * on the list.
+   **/
+  filterPlugins(results) {
```

Comment:
we can do that, but it won't actually matter. If the results is empty or even null, it will still return an empty array